packages/flutter/lib/src/widgets/localizations.dart

```diff
@@ -311,6 +323,49 @@ class Localizations extends StatefulWidget {
     assert(delegates.any((LocalizationsDelegate<dynamic> delegate) => delegate is LocalizationsDelegate<WidgetsLocalizations>));
   }
 
+  /// Overrides the inherited [Locale] or [LocalizationsDelegate]s for `child`.
+  ///
+  /// This factory constructor is used for the - usually rare - situtation where part
+  /// of an app should be localized for a different locale than the one defined
+  /// for the device, or a different list of [LocalizationsDelegate]s than the
+  /// list defined by [WidgetsApp.localizationsDelegates].
+  ///
+  /// For example you could specify that `myWidget` was only to be localized for
+  /// the US English locale:
+  /// ```
+  /// Widget build(BuildContext context) {
+  ///   return new Localizations.override(
+  ///     context: context,
+  ///     locale: const Locale('en', 'US'),
+  ///     child: myWidget,
+  ///   );
+  /// }
+  /// ```
+  ///
+  /// The `locale` and `delegates` parameters default to the [Localizations.locale]
+  /// and [Localizations.delegates] values from the nearest [Localizations] ancestor.
+  ///
+  /// To override the [Localizations.locale] or [Localizations.delegates] for an
+  /// entire app, specify [WidgetsApp.locale] or [WidgetsApp.localizationsDelegates]
+  /// (or specify the same parameters for [MateialApp]).
+  factory Localizations.override({
```

Comment:
I think we normally call this "merge" rather than "override" (consider `DefaultTextStyle.merge`).

Reply:
though maybe override is clearer in this case